src/components/settings.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+// Setting class having the profile details
+
+class Settings extends Component {
+
+    render(){
+
+        return (
+
+            <div>
+
+                <img src={'http://www.the-lizard.org/images/phocagallery/Mammals/Rabbit_320_Natural%20England_Allan%20Drewitt.jpg'} alt="profile picture" className="profileImage" />
```

Comment:
Could you please download the jpg file and save it locally under img. Thanks :)

Reply:
:+1: